### DISCUSSION TITLE
Midi/drum editor 'snap' drop-down lost a column?

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1155] [Team Posts]
Just wondering, did someone change that?
The drop-down used to be three columns of 'snap' values, now it's just two.
Are you sure two columns is enough? Will it be OK? What about the other older
 values in the missing column?
Or did a bug creep in there?

**Comment by spamatica** [comment]
Hmm, it sort of rings a bell that I saw something wierd in that area some days ago, but trying now it looks OK. So, don't know what might be amiss.

**Comment by terminator356** [comment]
Oh, I see now. The other column is there but not being displayed.
I must use the keyboard arrows to get it into view. Mouse alone does not show it.
Maybe with all the font changes recently, that drop-down list suffered something.
Likely its width needs to be adjusted.
We'll have to fix that...

**Comment by kybos** [comment]
It looks fine here, I haven't noticed anything weird.
AFAIK there were no relevant font changes here, the toolbar has always used the main system font?

**Comment by spamatica** [comment]
Haven't looked at the code but did a bit of headscratching where I saw it,
and it was on the computer where I have a more recent Kubuntu, meaning that
it very likely is a Qt thing. Went and started MusE there now and it only
shows two columns, so something changed in Qt...

Den lör 15 feb. 2020 kl 03:38 skrev Tim <notifications@github.com>:

**Comment by terminator356** [comment]
I think it's just our drop-down width setting.
I recall this also happened many years ago when I fixed it.
The width was not 'automatic' if I recall, that is, we must set it.
It may be a sort of fixed, or poorly calculated, value.

**Comment by terminator356** [comment]
OK I applied a fix.
Can you guys check it out? Especially kybos on his display.
There should be three 'snap' columns fitting snugly with no excess movement to either side.
Thanks!

**Comment by kybos** [comment]
I could reproduce the problem only on Manjaro/KDE in a VM.
With the fix it looks OK now also there.
